runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -391,7 +393,22 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 
 			if (config.showMenuItemQuantities() && itemComposition.isStackable() && quantity > 1)
 			{
-				lastEntry.setTarget(lastEntry.getTarget() + " (" + quantity + ")");
+				final String amount = quantity < GroundItemsOverlay.MAX_QUANTITY ? StackFormatter.quantityToStackSize(quantity) : "Lots!";
```

Comment:
Maybe common method that accepts StringBuilder and amout? to not introduce duplicated code

Reply:
I was looking at adding a function to the StackFormatter when I started but I wasn't sure it required it. A plugin-local one would make sense though.